command/node_meta_apply.go

```diff
@@ -0,0 +1,119 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/posener/complete"
+)
+
+type NodeMetaApplyCommand struct {
+	Meta
+}
+
+func (c *NodeMetaApplyCommand) Help() string {
+	helpText := `
+Usage: nomad node meta apply [-node-id ...] [-unset ...] key1=value1 ... kN=vN
+
+  Modify a node's metadata. This command only works on client agents, and can
```

Comment:
```suggestion
  Modify a node's metadata. This command only applies to client agents, and can
```

"only works on" sounds like you cannot run this command against a server (but you can, with -node-id, right?)

Reply:
`-node-id` only routes to client agents, but I like your wording better. Thanks!